packages/material-ui/src/locale/index.js

```diff
@@ -663,14 +716,18 @@ export const huHU = {
 };
 
 export const hyAM = {
+  localization: {
+    formatNumber: formatNumber('hy-AM'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@vgevorgyan Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).